### ISSUE TITLE
Reverse Geocode Syntax Error

### ISSUE THREAD

**Issue by madkatz01** [FrankBijnen/ExifToolGui#334]
When trying to reverse geocode, it appears that it formats the API request to geocode incorrectly. When I click "show on map" I get an error staying "Request failed with: Bad Request. Continue?"

Looking at the log, here is what I see the request as (I changed the lat/lon to all 1s and removed my API key):
https://geocode.maps.co/reverse?lat=11.1111111&lon=--11.111111111&api_key=myapikeyhere

And the response:: {"error":{"code":400,"message":"Floating-point number expected for parameter 'lon'"}}

Basically, there are two "-" in front of the longitude. If I remove one manually in a browser, I can complete the API request. Tried this with many different photos and all of them have "--" as part of the lon, correct would be one "-".

**Comment by FrankBijnen** [comment]
This is a known issue with V629. Has been fixed by: https://github.com/FrankBijnen/ExifToolGui/issues/310
Will be in the next release V6.3.0.
For a temporary fix you can download: https://1drv.ms/f/s!AhVJC-QzM33pknOS7KwBi09lgNql?e=n7ztFX

**Comment by Philshappy** [comment]
Tried it and it seems to work. Thanks!
